ansible/files/admin_api_scripts/disable_walg.sh

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+sed -i "s/.*archive_mode/#archive_mode/" /etc/postgresql/postgresql.conf
```

Comment:
Instead of modifying this conf file, what do you think of including a separate file, and writing to it/emptying it out to enable/disable wal-g?

Reply:
Updated to `include` a separate file housing parameters pertaining to wal-g. Scripts are pointed to this new file instead to modify these parameters.